### ISSUE TITLE
Doesn't really work on iPad 2 iOS7

### ISSUE THREAD

**Issue by mjurczyk** [Dogfalo/materialize#762]
Has some real issues on iPad 2, which is pretty popular after all and should be considered a still significant mobile device.

1. Clicking combo box arrow doesn't trigger the combo-box at all.
2. Pre-loaders' animations stop when you scroll page.
3. Parallax doesn't work at all ( I mean, ok, it works, but only after you end scrolling, which is bad ).
4. Side menu 'dragging' doesn't work.
5. Check-box tick is jagged, looks really nasty.
6. Clicking the menu in top-left opens it after about 3sec.
7. Side-nav doesn't continue with the page, unless you stop scrolling.

I'm not quite sure if any in-depth explanation is need for any of those.
Still, good job, it is fun on pc!

**Comment by carlosperate** [comment]
> 1 Clicking combo box arrow doesn't trigger the combo-box at all.

Are you referring to the the drop down buttons? If so, they work fine for me.

> 2 Pre-loaders' animations stop when you scroll page.

I would assume this behaviour is done by the browser to improve scrolling performance, I doubt there is much that can be done, or if it should be done at all.

(I have no comments about 3)

> 4 Side menu 'dragging' doesn't work.

That one doesn't work because it triggers the browser feature to switch tabs. You can actually see the menu being dragged slightly before the tab is switched.

> 5 Check-box tick is jagged, looks really nasty.

I don't really know what you mean, could you post a screenshot. I just tried it and it looks the same than in my laptop with chrome.

> 6 Clicking the menu in top-left opens it after about 3sec.

This might just be a performance issue, I've still got iOS7 in an ipad mini 2 and the response time is well within an acceptable range.

> 7 Side-nav doesn't continue with the page, unless you stop scrolling.

Could you elaborate about this? A quick check in the materializecss website and I do not see any expected behaviour with the side bar nor the navbar examples

**Comment by Dogfalo** [comment]
It would also be helpful to know what browser you are using

**Comment by mjurczyk** [comment]
Ok, lets fight :smile: Now, but seriously, first thing to note - I am talking **iPad2** not **iPad mini 2**, there is quite a significant difference (A5 vs A7).

K, and now back to the points:

1. I mean this little fella on the right-bottom here:
![screen shot 2015-02-26 at 15 44 13](https://cloud.githubusercontent.com/assets/9549760/6393815/88969ee4-bdcd-11e4-957c-495a2114d05c.png)
This arrow is clickable on Safari iOS7 ( I just noticed it is not clickable in Chrome, which is odd ), there is a click animation, but nothing happens. I would really assume that this arrow actually opens the combo-box for me ;)

2. Doesn't feel native, but k :/ Let's just give it up.

3. I have no comments regarding your lack of comments. My comment was greatly related to the point 7. actually. Remember that mobile browsers do not have scroll event, only scroll-end. [A little more on that topic](http://andyshora.com/mobile-scroll-event-problems.html).

4. Do you even prevent default, bruh? ( By the way, doesn't seem to work at all on Galaxy S2 Chrome ).

5. Sorry, can't really do a screenshot right now, I'll do that later if I have a second. Just to explain that - transforms do not apply antialiasing.

6. No comments from me, depends on how performant you want it to be.

7. This bundle of happiness:
![screen shot 2015-02-26 at 15 54 25](https://cloud.githubusercontent.com/assets/9549760/6394024/f1f43f30-bdce-11e4-9e58-ee5eb3b07ee5.png)
Doesn't follow the page as you scroll down on a tablet. This is related to point 3., scroll vs scrollend on mobile.

8. **(new one)**. Android Chrome, SG2. When I want to scroll down the side menu:
![screen shot 2015-02-26 at 15 58 34](https://cloud.githubusercontent.com/assets/9549760/6394113/856bd89a-bdcf-11e4-8da1-2a6d6743acb4.png)
And I start the dragging process by putting my finger on the closed section ( for example 'Components' ), UI automatically does this 'bubble' animation on the section button and closes the navigation.

@Dogfalo safari on iOS7 on iPad and Android Chrome on SG2.

**Comment by carlosperate** [comment]
I am taking a few days off and won't be able to this this again until next week (might be able to reply if I find some free time), but I just quickly wanted to add that for the following:

> I am talking iPad2 not iPad mini 2, there is quite a significant difference (A5 vs A7).

Sorry, I didn't mean to insinuate I was testing on the same hardware as you, just that the OS was the same, and I would assume it should behave similarly with the exception of performance.

> 1 I mean this little fella on the right-bottom here:
This arrow is clickable on Safari iOS7 ( I just noticed it is not clickable in Chrome, which is odd ), there is a click animation, but nothing happens. I would really assume that this arrow actually opens the combo-box for me ;)

For me as well the arrow is indeed 'clickable', in the sense that it briefly highlights on touch, but it does nothing.
Also should add that when the dropdown select menu is deployed, it does not close after clicking anywhere else, I need to either select an option or a different drop down for it to close.

> Do you even prevent default, bruh? ( By the way, doesn't seem to work at all on Galaxy S2 Chrome ).
I have tested it in several android devices (including and s4 mini) on Android 4.0.x and kitkat in chrome and it has worked fine for me so far, maybe a bit more info about your test environment could shed some light onto your issue. 

> 7 This bundle of happiness:
Doesn't follow the page as you scroll down on a tablet. This is related to point 3., scroll vs scrollend on mobile.

Yep, indeed not working.

> 8 (new one). Android Chrome, SG2. When I want to scroll down the side menu:
And I start the dragging process by putting my finger on the closed section ( for example 'Components' ), UI automatically does this 'bubble' animation on the section button and closes the navigation.

Yep, can confirm issue.

**Comment by Dogfalo** [comment]
8. Fixed

**Comment by Dogfalo** [comment]
Made a separate issue for 8. The rest are either already existing issues or issues related to a extremely low performance device. 4,5,6 have not been reported by anyone else.

**Comment by lexbi** [comment]
RE Parallax, ipad2, and iphone 4, both on ios7, still behave like OP suggests, the position changes after scrolling has finished. Iphone 6, ipad pro, ipad3 all seem ok on ios 9 and above.
